test/core/RandomVariables.jl

```diff
@@ -190,41 +231,54 @@ include("../test_utils/AllUtils.jl")
         @test cuid(vn1) == cuid(vn2)
         @test vn11 == vn1
 
-        vi = VarInfo()
         dists = [Normal(0, 1), MvNormal([0; 0], [1.0 0; 0 1.0]), Wishart(7, [1 0.5; 0.5 1])]
+        function test_varinfo!(vi)
```

Comment:
Would it be nice to improve the tests here a bit, e.g. adding more comments on what's testing and expected, and improving the code (variable names, code reuse, etc).

Reply:
I will add more comments to the src too.